WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -155,6 +166,27 @@ class WPRichContentView: UITextView {
         }
     }
 
+    private func setupTouchDetection() {
+          for gesture in gestureRecognizers ?? [] {
+              gesture.require(toFail: linkTapGestureRecognizer)
```

Comment:
I thought this would be necessary but I commented it out and things still seem to work (after implementing the `shouldReceive` method). What was this trying to prevent?

Also, it seems to be interfering with the normal behavior of the text view. When I long press on a link, the share sheet pops up now ❤️ but the word I tapped also gets selected.

![Simulator Screen Shot - iPhone 11 - 2019-10-03 at 09 37 57](https://user-images.githubusercontent.com/8739/66107861-943d3980-e5c1-11e9-954b-385ba52dbf05.png)

Reply:
You're right! Removing it